pkg/plan/resource/kubeadm_init.go

```diff
@@ -253,7 +257,8 @@ func buildKubeadmInitPlan(path string, ignorePreflightErrors string, useIPTables
 		// certificates of the primary control plane in the kubeadm-certs
 		// Secret, and prints the value for --certificate-key to STDOUT.
 		&Run{Script: plan.ParamString(
-			withoutProxy("kubeadm init --config=%s --ignore-preflight-errors=%s --experimental-upload-certs"), &path, &ignorePreflightErrors),
+			withoutProxy("kubeadm init --config=%s --ignore-preflight-errors=%s --experimental-upload-certs"),
```

Comment:
From the documentation I only see --experimental-upload-certs available in 1.14.  Have you tested on other versions?

Reply:
we currently allow to start the cluster with only k8s 1.14.1.
I'll fix this in another PR.